spec/facets/ColumnLineageDatasetFacet.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://openlineage.io/spec/facets/1-1-0/ColumnLineageDatasetFacet.json",
+  "$id": "https://openlineage.io/spec/facets/1-2-0/ColumnLineageDatasetFacet.json",
```

Comment:
Spec changes should be done in separate PRs

Reply:
https://github.com/OpenLineage/OpenLineage/pull/3098